lib/generator.ts

```diff
@@ -16,13 +16,10 @@ var Environment = (function() {
         paths.pop();
 
         _.each(getAllConsumingPackages(), x => paths.push(x));
+
         // add the default path for the user.
-        if (process.platform === 'win32') {
-            paths.push(path.join(process.env.APPDATA, 'npm/node_modules'));
-        } else {
```

Comment:
How about we leave these paths in, but then additionally add the global paths?  If the folders don't exist, then they're ignored anyway.

There may still be issues for some other edge cases, but this will move us closer.